### DISCUSSION TITLE
how to add everybody in /accestrust & /containertrust ?

### DISCUSSION THREAD

**Discussion by Vik4o0** [GriefPrevention/GriefPrevention#1717] [Q&A] [answered]
how to add everybody in /accestrust & /containertrust ? I tried to make a chest on spawn with useless stuff so players can donate to the newbies. So i want to trust everyone to access opening the chest. How to do that?

**Comment by Jikoo** [comment, marked as answer]
You can use `/accesstrust public` or `/accesstrust all` to select everyone. These are functionally the same for adding trust, but for removing it `all` removes everyone including specifically trusted players, where `public` only removes the general public's trust.
